browser/src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Ya. I think for our requirements, It'll be fine to have `index.html` sit in `/public`. If we start doing some more advanced stuff with webpack, like caching or code-splitting, or stuff like that, then I'll probably have to add html-webpack-plugin back in and have a index.html template sitting somewhere in our project.

Reply:
That's a good point - something to note for the future!

Reply:
Oh interesting, http://localhost:3001/ gives me this in the browser: `Cannot GET /`.

It looks like, because we exclude `/public` from the source, we no longer have an index file. Because we will want to store other static assets such as images and fonts in source, let's change the gitignore rule of `/public` to _just_ the JS bundle. Then, you can submit the index.html to source. Just a note, eventually we'll want a build folder. When we go down this route, we can take a look at how create-react-app does it.

Side note, do we still need `/public/css`?

Reply:
Thinking about this more, let's take advantage of something like https://webpack.js.org/plugins/html-webpack-plugin/.

Let's:
1) Rename `/browser/public` to `/browser/assets`
2) Create a new `/browser/dist` folder. Bundles will go here.
3) We can get rid of `/browser/public/css`.
4) Do the macarena.

Reply:
Ha. Nobody should have to witness me do the macarena, but I will take care of everything else. Thanks.

Reply:
I'm curious, is this working on your machine? It looks like we're missing an `index.html` file and I don't see anything being generated in a `/dist` folder.

Reply:
index.html won't be generated unless there is a `webpack build`. `webpack-dev-server` will create the js bundle and the html file in memory, then will immediately destroy it once the server is disconnected.